lib/ssh/src/ssh_client_channel.erl

```diff
@@ -248,8 +244,6 @@ call(ChannelPid, Msg, TimeOute) ->
     end.
 
 -doc """
-cast(ChannelRef, Msg) -> ok
-
 Sends an asynchronous message to the channel process and returns ok immediately,
```

Comment:
pls explain. is leaving paragraph containing more than 1 sentence as a signature also fine? In other places I thought single sentence is preferred ...